components/imageproc/src/lib.rs

```diff
@@ -343,6 +346,35 @@ impl ImageOp {
     }
 }
 
+/// Apply image rotation based on EXIF data
+pub fn fix_orientation(img: DynamicImage, path: &Path) -> DynamicImage {
+    let file = std::fs::File::open(path).unwrap();
+    let mut buf_reader = std::io::BufReader::new(&file);
+    let exif_reader = exif::Reader::new();
+    let exif = exif_reader.read_from_container(&mut buf_reader);
+    let exif = match exif {
```

Comment:
you can put ` exif_reader.read_from_container(&mut buf_reader)` after `match` and you avoid one variable named exif